src/pages/SetPassword.vue

```diff
@@ -4,6 +4,10 @@
     title="New Account 2/2"
     back-button
   >
+    <h1 class="width-254 size-35">Create a password</h1>
```

Comment:
I think here should be used an explicit line break (`<br />`) before "password" instead of `max-width: 254px`. Because this header should be styled in the same way in other pages, and other pages have wider titles.